gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -176,11 +176,21 @@ class GateConfig extends RedisHttpSessionConfiguration {
   }
 
   @Bean
+  @Primary
   FiatService fiatService(OkHttpClient okHttpClient) {
     // always create the fiat service even if 'services.fiat.enabled' is 'false' (it can be enabled dynamically)
     createClient "fiat", FiatService, okHttpClient, null, true
   }
 
+  @Bean
+  @Qualifier("fiat-login")
```

Comment:
Isn't this basically the same as if we named the bean factory method `fiatLogin` (assuming we dropped the dash)?

Reply:
honestly I'm not 100% sure how all the rules apply in the face of an optional bean so I went with the most explicit route I could

Reply:
removed the qualifier on here and everything still works as expected